IopHomeNode.proto3

```diff
@@ -433,12 +433,12 @@ message ServerRole {
 //      |                                                                     |
 //      |  StartConversationRequest                                           |
 //      *-------------------------------------------------------------------> *
-//      |                                                                     |
+//      |  <clientChallenge>                                                  |
 //      |                                                                     | status := ConversationStarted
 //      |                                                                     |
 //      |                                          StartConversationResponse  |
 //      * <-------------------------------------------------------------------*
-//      |                                                        <challenge>  |
+//      |                                <challenge, signed clientChallenge>  |
```

Comment:
In some cases it is, in other cases only a specific part is being signed. This is message specific now.